spec/features_spec.rb

```diff
@@ -0,0 +1,207 @@
+require 'score'
+
+describe 'Bowling scorecard features' do
+  let(:score) { Score.new }
+
+  it 'can store the score of a single roll and display it for the user' do
+    score.record_roll(5)
+    expect(score.check_current_frame).to eq([5])
+  end
+
+  it 'can display the correct score after multiple frames (no strikes or spares)' do
+    score.record_roll(5)
+    score.record_roll(2)
+    score.record_roll(4)
+    score.record_roll(1)
+    score.record_roll(3)
+    score.record_roll(6)
+    expect(score.calculate_score).to eq(21)
+  end
+
+  it 'can display the correct score over multiple frames (including a spare)' do
+    score.record_roll(5)
+    score.record_roll(2)
+    score.record_roll(7)
+    score.record_roll(3)
+    score.record_roll(3)
+    score.record_roll(6)
+    expect(score.calculate_score).to eq(29)
+  end
+
+  it 'can display the correct score over multiple frames (including multiple spares)' do
+    score.record_roll(5)
+    score.record_roll(2)
+    score.record_roll(7)
+    score.record_roll(3)
+    score.record_roll(3)
+    score.record_roll(7)
+    score.record_roll(2)
+    score.record_roll(1)
+    expect(score.calculate_score).to eq(35)
+  end
+
+  it 'can display the correct score over multiple frames (including a strike)' do
+    score.record_roll(5)
+    score.record_roll(2)
+    score.record_roll(7)
+    score.record_roll(3)
+    score.record_roll(3)
+    score.record_roll(7)
+    score.record_roll(10)
+    score.record_roll(2)
+    score.record_roll(1)
+    expect(score.calculate_score).to eq(56)
+  end
+
+  it 'can correctly score a ten-frame score (no strike or spare on final frame)' do
+    score.record_roll(5)
+    score.record_roll(2)
+    score.record_roll(7)
+    score.record_roll(3)
+    score.record_roll(3)
+    score.record_roll(7)
+    score.record_roll(10)
+    score.record_roll(2)
+    score.record_roll(1)
+    score.record_roll(6)
+    score.record_roll(1)
+    score.record_roll(2)
+    score.record_roll(4)
+    score.record_roll(3)
+    score.record_roll(7)
+    score.record_roll(10)
+    score.record_roll(2)
+    score.record_roll(1)
+    expect(score.calculate_score).to eq(105)
+  end
+
+  it 'can correctly score a full score with spare on final frame' do
+    score.record_roll(2)
+    score.record_roll(2)
+    score.record_roll(2)
+    score.record_roll(2)
+    score.record_roll(2)
+    score.record_roll(2)
+    score.record_roll(2)
+    score.record_roll(2)
+    score.record_roll(2)
+    score.record_roll(2)
+    score.record_roll(2)
+    score.record_roll(2)
+    score.record_roll(2)
+    score.record_roll(2)
+    score.record_roll(2)
+    score.record_roll(2)
+    score.record_roll(2)
+    score.record_roll(2)
```

Comment:
you can do something like 18.times { score.record_roll(2) } (I'm not sure that's quite right but something like that) to save yourself a bunch of lines :)